core/src/main/kotlin/org/evomaster/core/output/naming/RPCActionTestCaseNamingStrategy.kt

```diff
@@ -0,0 +1,40 @@
+package org.evomaster.core.output.naming
+
+import org.evomaster.core.output.TestWriterUtils
+import org.evomaster.core.problem.rpc.RPCCallAction
+import org.evomaster.core.problem.rpc.RPCCallResult
+import org.evomaster.core.search.EvaluatedIndividual
+import org.evomaster.core.search.Solution
+import org.evomaster.core.search.action.EvaluatedAction
+
+open class RPCActionTestCaseNamingStrategy(
+    solution: Solution<*>,
+    languageConventionFormatter: LanguageConventionFormatter
+) : ActionTestCaseNamingStrategy(solution, languageConventionFormatter)  {
+
+    override fun expandName(individual: EvaluatedIndividual<*>, nameTokens: MutableList<String>): String {
+        var evaluatedAction = individual.evaluatedMainActions().last()
+        var action = evaluatedAction.action as RPCCallAction
+
+//        nameTokens.add(TestWriterUtils.safeVariableName(action.interfaceId))
```

Comment:
Hi @arcuri82, got doubts here.

I've executed the [BranchesGRPCEMTest](https://github.com/WebFuzzing/EvoMaster/blob/master/e2e-tests/spring-rpc/spring-rpc-grpc/src/test/java/org/evomaster/e2etests/spring/rpc/grpc/examples/branches/BranchesGRPCEMTest.java) test and checked the generated test file. Despite the fact that it's just a test class, results were as follows:
- Using both `action.interfaceId` and `action.id`: `test_5_com_foo_rpc_grpc_examples_spring_branches_generated_branchgrpcservicegrpc_branchgrpcserviceblockingstubCom_foo_rpc_grpc_examples_spring_branches_generated_branchgrpcservicegrpc_branchgrpcserviceblockingstub_negReturnsSuccess` (231 chars long)
- Using just `action.id`: `test_5_com_foo_rpc_grpc_examples_spring_branches_generated_branchgrpcservicegrpc_branchgrpcserviceblockingstub_negReturnsSuccess` (128 chars long)

There seems to be an awful lot of duplicated (and long) information. Checking the [branches.proto](https://github.com/WebFuzzing/EvoMaster/blob/master/e2e-tests/spring-rpc/spring-rpc-grpc/src/main/proto/branches.proto) file I can see the service (`BranchGRPCService`) and operations (`pos`, `neg`, `eq`) are very well defined and nearly not as half as long. Maybe instead of using the whole `action.interfaceId` and `action.id` we could try to extract the service and operation name? I don't know much about RPC, maybe I'm missing something

Reply:
hi @Pgarrett  you right. ideally for an interface/class, we don't  need its full name with package, but just its simple name,eg, `simpleclassname.methodname`
might need to do some modifications in `RPCCallAction` to get that info reliably

Reply:
Ok @arcuri82, I'll look into it and see what I can find. Thanks!